package.json

```diff
@@ -2,7 +2,7 @@
   "name": "cosmy-wasmy",
   "displayName": "Cosmy Wasmy",
   "description": "wibbly wobbly cosmy wasmy tool to interact with cosmwasm smart contracts",
-  "version": "2.3.2",
+  "version": "2.3.2+finschia",
```

Comment:
How about set `2.3.1+finschia`?

Reply:
Because we are working after this commit : https://github.com/spoo-bar/cosmy-wasmy/commit/0ea99ecea65e51e197ac8ecb3693c9377b1f1563, so we've included some features from version 2.3.2.

Reply:
But `2.3.2` is not released. So...

Reply:
Thank you very much! I will fix it.